packages/react/.storybook/_container.scss

```diff
@@ -1,9 +1,88 @@
 $css--font-face: true;
+$css--helpers: true;
+$css--body: true;
+$css--use-layer: true;
 $css--reset: true;
+$css--plex: true;
 
 $prefix: 'bx';
 
-// IMPORTANT: This import path should _not_ be used outside our source tree
-// as `src` directory is _not_ meant to be shipped in our NPM package.
-// Use e.g. `@import '~carbon-components/scss/globals/scss/styles.scss'` instead.
-@import '~carbon-components/src/globals/scss/styles.scss'; // SEE THE NOTE ABOVE
+@import '~carbon-components/src/globals/scss/feature-flags';
```

Comment:
Instead of maintaining the component-level token usage pattern here, would we want to do the two?

- Writing some tests
- Documentation

Reply:
I don't follow. Changes to `_container.scss` are just for demonstration purposes only as specified in the RFC description. I don't intend on this PR getting merged with any `_container.scss` changes.

Reply:
Your RFC description said it's for demonration, but didn't know that it's not meant for merge - Thank you for clarifying @mattrosno!